app/models/game.rb

```diff
@@ -112,18 +119,43 @@ def self.search(title, publisher, tourney, checked, loaned, group)
       result = result.includes(:loans).references(:loans).merge(Loan.where(closed: false, group: group, event: Event.current))
     end
 
-    result.where(culled: false)
+    result.where(status: 0)
   end
 
   def cull_game
     if self.checkouts.where(closed: false).size == 0
-      self.update(culled: true)
+      self.update(status: 1)
       'Game successfully culled!'
     else
       'Game is still checked out! Please return game before culling it.'
     end
   end
 
+  def store_game
+    if self.checked_out?
+      return {
+        error: true,
+        message: 'Game is currently checked out by an attendee and can not be stored!'
+      }
+    end
+
+    if self.status == "storage"
+      self.update(status: 0)
```

Comment:
If status is "storage" why are you updating it to an integer? Wouldn't the status be either an integer or a string, but not both?

Reply:
Status is an integer in the database, but I added an enum to Game that translates those to strings. self.status gives it back as a string, so that's how I check for equality. I might be able to pass in the string to the update method as well? Will test.

Reply:
Was able to pass a string into the update method! Thanks!